science/gerris/Portfile

```diff
@@ -1,61 +1,74 @@
-PortSystem 1.0
+#-*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-name		gerris
-version		0.9.2
-categories	science
-license		GPL-2+
-platforms	darwin
-maintainers	nomaintainer
-description	The Gerris Flow Solver
-long_description	\
-		Gerris is an Open Source Free Software library		\
-		for the solution of the partial differential		\
-		equations describing fluid flow.
+PortSystem            1.0
+PortGroup             mpi 1.0
+PortGroup             compiler_blacklist_versions 1.0
 
-homepage	http://gfs.sourceforge.net/
-master_sites	sourceforge:gfs
+name		          gerris
+version		          131206
```

Comment:
Same comment about `version` here as with gts: Use an 8-digit `YYYYMMDD` date, and if this snapshot identifies itself as a particular real version number, use that in the `version` field as well.

Reply:
This should now be fixed.